internal/dashboard/static/style.css

```
@@ -1,170 +1,458 @@
-  max-width: 1200px;
+  max-width: 1080px;
   margin: 0 auto;
-  padding: 16px 24px;
+  padding: 24px 32px 48px;
+  line-height: 1.5;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+/* ── page load animation ── */
+@keyframes fadeUp {
```

Stylelint (17.3.0)

[error] Expected keyframe name "fadeUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)